cluster/internal/data.pb.go

```diff
@@ -17,15 +17,17 @@ It has these top-level messages:
 package internal
 
 import proto "github.com/gogo/protobuf/proto"
+import fmt "fmt"
 import math "math"
 
 // Reference imports to suppress errors if they are not otherwise used.
 var _ = proto.Marshal
+var _ = fmt.Errorf
 var _ = math.Inf
 
 type WriteShardRequest struct {
-	ShardID          *uint64  `protobuf:"varint,1,req" json:"ShardID,omitempty"`
-	Points           [][]byte `protobuf:"bytes,2,rep" json:"Points,omitempty"`
+	ShardID          *uint64  `protobuf:"varint,1,req,name=ShardID" json:"ShardID,omitempty"`
```

Comment:
Ah, ok.  This still comes down to a vendoring issue though.  Do you normally run go generate just on the specific package you changed?  I don't think that will get us into trouble, but certainly we will keep seeing code churn every time you generate vs me.  I guess what I'm saying is I don't disagree with you, but the issue is beyond the scope of this PR, and it is more about vendor version management than anything else.